src/datatypes/authentication/ChangePassword.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Define type for Changing Password Form.
+ * Validator also implemented
+ *
+ * @author Hyecheol (Jerry) Jang <hyecheol123@gmail.com>
+ */
+
+/**
+ * Interface for ChangePasswordForm
+ */
+export default interface ChangePasswordForm {
+  currentPassword: string;
+  newPassword: string;
```

Comment:
new Password double-check??
As our password policy is quite hard, i think we really need it

Reply:
This is the "Interface", indicating the datatype of password change form. As typescript only checks data type during compile time, we need to implement a separated runtime type checker.